components/atom/progressBar/README.md

```diff
@@ -0,0 +1,38 @@
+# AtomProgressBar
+
+AtomProgressBar is a component that displays a progress bar
```

Comment:
what about `A Progress bar is a visual representation of a specified wait time or the current status of a process.`?